src/generator/analysis/__init__.py

```
@@ -2,11 +2,12 @@
-- AnalysisTransformer: Extracts entities from HBF and clusters them by type
+- RawEntities: Main orchestration model for entity analysis
+- AnalysisSummary: Summary of analysis results
 - Constants: Entity lists and configuration
 """
 
-from generator.analysis.transformer import AnalysisTransformer
+from generator.analysis.models import RawEntities, AnalysisSummary
 from generator.analysis.constants import REGIONS, SETTLEMENTS, FACTIONS, DUNGEONS
 
-__all__ = ["AnalysisTransformer", "REGIONS", "SETTLEMENTS", "FACTIONS", "DUNGEONS"]
+__all__ = ["RawEntities", "AnalysisSummary", "REGIONS", "SETTLEMENTS", "FACTIONS", "DUNGEONS"]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)